docs/in-depth-guide-scraper-functions.md

```diff
@@ -322,3 +322,12 @@ Returns the method of cooking the recipe.
 >>> scraper.cooking_method()
 'Stovetop'
 ```
+
+### `dietary_restrictions"() -> List[str] | None`
+
+Returns the dietary restrictions specified by the recipe.
+
+```py
+>>> scraper.equipment()
+['Vegan Diet', 'Vegetrian Diet']
```

Comment:
```suggestion
>>> scraper.dietary_restrictions()
['Vegan Diet', 'Vegetrian Diet']
```

Reply:
Should the spelling of `Vegetrian` -> `Vegetarian` be fixed too?

Reply:
Nice catch!  Yep, it's not in the [`schema.org` enumeration](https://schema.org/RestrictedDiet), so I'll edit my suggestion to fix that too.